network-protection/network-protection-store/src/main/java/com/duckduckgo/networkprotection/store/waitlist/NetPWaitlistRepository.kt

```diff
@@ -14,19 +14,14 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.networkprotection.impl.waitlist.store
+package com.duckduckgo.networkprotection.store.waitlist
 
 import com.duckduckgo.app.global.DispatcherProvider
-import com.duckduckgo.di.scopes.AppScope
-import com.duckduckgo.networkprotection.impl.state.NetPFeatureRemover
-import com.squareup.anvil.annotations.ContributesBinding
-import com.squareup.anvil.annotations.ContributesMultibinding
-import javax.inject.Inject
 import kotlinx.coroutines.withContext
 
 interface NetPWaitlistRepository {
     fun getAuthenticationToken(): String?
-    fun setAuthenticationToken(authToken: String)
+    fun setAuthenticationToken(authToken: String?)
```

Comment:
Is this change because you need to `null` the auth token somewhere?
In what case we need to `null` it?

Reply:
Ah. This is unintentional. Will revert.